### DISCUSSION TITLE
OSError: [Errno 22] Invalid argument: '.server_status_sys_id'

### DISCUSSION THREAD

**Discussion by FASUZO** [zdz/ServerStatus-Rust#217] [Q&A] [answered]
<img width="625" alt="uTools_1685931595556" src="https://github.com/zdz/ServerStatus-Rust/assets/77849230/7e8eb009-568b-4af3-8c84-9c1ac94115ff">

**Comment by FASUZO** [comment]
<img width="631" alt="uTools_1685931819637" src="https://github.com/zdz/ServerStatus-Rust/assets/77849230/0770a5bd-4bf6-4821-bc09-b3e185756b27">

我技术水平太低，并未发现源代码哪里有问题

**Comment by zdz** [comment]
什么系统

**Reply by FASUZO** [reply]
Win10 2016-LTSC

**Reply by FASUZO** [reply]
![image](https://github.com/zdz/ServerStatus-Rust/assets/77849230/421e6a5d-0990-426b-a56e-9550f2ebc7de)

**Comment by zdz** [comment]
可以先试试把代码里面的 `.server_status_sys_id` 改成 `abc` 试试

**Reply by FASUZO** [reply]
![image](https://github.com/zdz/ServerStatus-Rust/assets/77849230/2afb94d8-4ba8-4a65-9bdc-b3e2dea8e2a9)

**Reply by zdz** [reply]
额，没这环境没办法弄

**Reply by FASUZO** [reply]
嗯，我也很奇怪，我本机windows就没问题一样的 py版本和源脚本

**Comment by zdz** [comment]
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/381d1d05-117b-4b4e-8264-ecf6f391621e)
可以在这台机上面把黄框部分删掉，影响不大

**Reply by FASUZO** [reply, marked as answer]
删除，已解决，返回值200

不过网站上没有出现https://list.suzo.top/

另外已排查出原始问题，Win LTSC远控权限不足，无法打开所创建的文件，所以报OSError: [Errno 22] Invalid argument错误，是我和系统的问题。

**Reply by zdz** [reply]
居然不是报 `PermissionError: [Errno 13] Permission denied` ... , 看看是不是 sys id 重复了，被后面的主机用一样的 sys id 覆盖了

**Reply by FASUZO** [reply]
有可能，远控还没玩懂，不过修改-保存文件会报 `PermissionError: [Errno 13] Permission denied `
感谢作者解惑
